.github/CODEOWNERS

```diff
@@ -4557,7 +4557,7 @@
 /types/webvr-api/                               @efokschaner
 /types/week/                                    @sindrenm
 /types/weighted/                                @ccitro
-/types/weixin-app/                              @taoqf @AlexStacker
+/types/weixin-app/                              @taoqf @AlexStacker @Jimexist
```

Comment:
It seems you don't need add your name here, add it into index.d.ts:line5. 
you may need read this https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package

Reply:
@taoqf thanks, updated